Sources/TuistGenerator/Generator/DerivedFileGenerator.swift

```diff
@@ -139,3 +175,19 @@ final class DerivedFileGenerator: DerivedFileGenerating {
             .appending(component: "\(target.name).plist")
     }
 }
+
+private extension GeneratedSideEffect {
```

Comment:
What about moving this to the definition of `GeneratedSideEffect`. I believe this might be handy from other files too.